### PR TITLE
DT-2208: 🐛 Fix audit failure

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: convert-hrtime.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const convertHrtime = hrtime => {
```
Comment (petergphillips) [inline convert-hrtime.js:1]: couldn't work out how to get the module to play nicely with jest.

The implementation is
```
export default function convertHrtime(hrtime) {
	const nanoseconds = hrtime;
	const number = Number(nanoseconds);
	const milliseconds = number / 1000000;
	const seconds = number / 1000000000;

	return {
		seconds,
		milliseconds,
		nanoseconds
	};
}
```
but jest didn't seem to like the `export default`, giving error `SyntaxError: Unexpected token 'export'`

File: package.json [hunk 2]
```diff
@@ -8,23 +8,35 @@
   },
   "scripts": {
     "start": "node server.js",
-    "test": "mocha",
+    "test": "jest",
```
Comment (petergphillips) [inline package.json:11]: should be using jest these days instead

File: proxy.test.js [hunk 3]
```diff
@@ -3,18 +3,18 @@ const express = require('express');
...
-const proxy = require('../proxy');
+const proxy = require('./proxy');
 
 const proxyApp = express();
 proxyApp.use(proxy);
 const request = supertest(proxyApp);
 
 describe('request proxying', () => {
-  before(() => {
+  beforeAll(() => {
```
Comment (petergphillips) [inline proxy.test.js:13]: jest equivalent of mocha method

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**